src/tools/metainfo/mdinfo.cpp

```diff
@@ -1203,8 +1212,10 @@ void MDInfo::DisplayMethodInfo(mdMethodDef inMethod, DWORD *pflags)
     if (!*sFlags)
         strcpy_s(sFlags, STRING_BUFFER_LEN, "[none]");
 
-    if (IsMdInstanceInitializerW(flags, memberName)) strcat_s(sFlags, STRING_BUFFER_LEN, "[.ctor] ");
-    if (IsMdClassConstructorW(flags, memberName)) strcat_s(sFlags,STRING_BUFFER_LEN, "[.cctor] ");
+    bool result = (((flags) & mdRTSpecialName) && !wcscmp((memberName), W(".ctor")));
```

Comment:
IsMdClassConstructorW does not have the proper Unicode string, and it is in WinSDK header file so it is not easy to fix it...